extension/src/options/index.js

```diff
@@ -58,95 +58,113 @@ function App() {
         <a href="https://prettier.io/docs/en/options.html">Prettier Options</a>
       </h1>
       <hr />
-      <label>Print width</label>
-      <input
-        type="number"
-        name="printWidth"
-        value={options.printWidth}
-        onChange={handleChange}
-      />
+      <label>
```

Comment:
For my own learning, how is this better than using the `for` attribute?

Reply:
Moving inputs inside labels automatically associates them in some browsers and accessibility tools. However in theory it's better to do both, if you prefer.

Reply:
As long as it works for Chrome, I think we're good :)